src/styles/app/37-database.css

```
@@ -941,49 +944,62 @@
 	background: linear-gradient(
 		180deg,
 		color-mix(in srgb, var(--interactive-accent) 8%, var(--bg-primary)) 0%,
 		color-mix(in srgb, var(--interactive-accent) 4%, var(--bg-secondary)) 100%
 	);
+	transform: translateY(-1px);
 }
 
 .databaseBoardCardTitle {
+	flex: 1 1 auto;
 	min-width: 0;
 	display: block;
```

Stylelint (17.9.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)